apps-rendering/src/components/liveBlocks.tsx

```diff
@@ -0,0 +1,97 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import { FirstPublished } from '@guardian/common-rendering/src/components/FirstPublished';
+import { LastUpdated } from '@guardian/common-rendering/src/components/LastUpdated';
+import LiveBlockContainer from '@guardian/common-rendering/src/components/liveBlockContainer';
+import { headline, space } from '@guardian/source-foundations';
+import { OptionKind, partition } from '@guardian/types';
+import type { DeadBlog, LiveBlog } from 'item';
+import type { FC } from 'react';
+import { renderAll } from 'renderer';
+
+// ----- Component ----- //
+interface LiveBlocksProps {
+	item: DeadBlog | LiveBlog;
+}
+
+interface BlockTitleProps {
+	title: string;
+}
+
+const BlockTitle: FC<BlockTitleProps> = ({ title }) => {
+	return (
+		<h2
+			css={css`
+				${headline.xxsmall({ fontWeight: 'bold' })}
+				margin-bottom: ${space[2]}px;
+			`}
+		>
+			{title}
+		</h2>
+	);
+};
+
+const LiveBlocks: FC<LiveBlocksProps> = ({ item }) => {
+	return (
+		<div>
```

Comment:
It could (I think). However, we made this a `div` because we think we may add some additional attributes to this root container for all liveblocks. The attributes (considering what's been done for dcr) could be: tabIndex (if we consider accessibility in the same way for mobile as we do for web?), an id of "maincontent" and potentially some global styles

Given the rationale above, do you think we could leave as a div for now (at least until we've considered the styling of liveblocks)?

Reply:
> if we consider accessibility in the same way for mobile as we do for web?
I think we should, yes.

> Given the rationale above, do you think we could leave as a div for now
In general I'd be inclined to leave it as a fragment until you're doing the work to add attributes, and change it to a `div` when you know you need it. Just in case you don't end up doing that work and it ends up staying as a `div` with nothing attached. But if you're confident that it will change then I don't think it matters too much 🙂

Reply:
Cool - have changed to a fragment for now!